.github/workflows/verify-workflow.yaml

```diff
@@ -0,0 +1,47 @@
+name: Verify
+
+# Controls when the action will run.
+on:
+  workflow_call:
+    secrets:
+      IBMCLOUD_API_KEY:
+        required: true
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  verify:
+    if: ${{ !contains( github.event.pull_request.labels.*.name, 'skip ci' ) }}
+    runs-on: ubuntu-latest
+    container: quay.io/cloudnativetoolkit/cli-tools-ibmcloud:v1.1
+
+    strategy:
+      matrix:
+        testcase:
+          - ocp4_latest
```

Comment:
This value needs to be vpc-ssh-generate and vpc-ssh-provided for the test to pass. i.e. the way it was in the original test

Reply:
Sure I am adding the Platform: to matrix
          - vpc-ssh-generate
          - vpc-ssh-provided